Formula/gnutls.rb

```diff
@@ -20,6 +21,7 @@ class Gnutls < Formula
   depends_on "nettle"
   depends_on "guile" => :optional
   depends_on "unbound" => :optional
+  depends_on "p11-kit"
```

Comment:
Aye, for now. We can review the situation later but if we introduce this by default and then end up removing `p11-kit` down the line because we discover further issues we'll have to force a rebuild of a wider group of formulae, which is a pain.